google-cloud-firestore/src/test/java/com/google/cloud/firestore/BulkWriterTest.java

```diff
@@ -158,7 +158,7 @@ public ScheduledFuture<?> schedule(Runnable command, long delay, TimeUnit unit)
 
     bulkWriter =
         firestoreMock.bulkWriter(BulkWriterOptions.builder().setExecutor(timeoutExecutor).build());
-    bulkWriter.autoShutdownBulkWriterExecutor = true;
```

Comment:
Should not have shut down timeoutExecutor, since that is always shut down in the `@After` method